wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -617,7 +624,15 @@ public DFUFileDetailInfo getFileDetails(String logicalname, String clustername)
             }
             this.handleException(resp.getExceptions());
             DFUFileDetailInfo info = new DFUFileDetailInfo(resp.getFileDetail());
-            info.setFirstline(this.getFirstRow(logicalname, clustername));
+            try
+            {
+                info.setFirstline(this.getFirstRow(logicalname, clustername));
```

Comment:
The first line of data for the file, used for determining the field count for sprayed csv files